.github/workflows/mbsolr-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: MusicBrainz Solr Packaging Tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Set up Eclipse Temurin 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Install brainz-mmd2-jaxb package with Maven
+        run: |
+          find src/main/java -type f -print0 | xargs -0 touch
```

Comment:
What's the purpose of this command?

Reply:
See the commit message in https://github.com/metabrainz/mb-solr/pull/55/commits/e501dae611e978cc3e25491bd2b17c6f072fe9b2

Reply:
See the message of commit https://github.com/metabrainz/mb-solr/pull/55/commits/e501dae611e978cc3e25491bd2b17c6f072fe9b2.

Reply:
Makes sense, Thanks!